cli/internal/typer/generator/platforms/kotlin/escape.go

```diff
@@ -23,6 +24,8 @@ func EscapeKotlinStringLiteral(s string) string {
 			builder.WriteString(`\\`)
```

Comment:
```suggestion
			builder.WriteString(`\\\\`)
```
Is escaping not needed here with the builder?